spec/unit/plugins/aix/virtualization_spec.rb

```diff
@@ -33,6 +251,56 @@
       expect(plugin[:virtualization][:lpar_no]).to eq("29")
       expect(plugin[:virtualization][:lpar_name]).to eq("l273pp027")
     end
+
+    context 'when WPARs exist on the LPAR' do
+      before do
+        @plugin = get_plugin("aix/virtualization")
+        allow(@plugin).to receive(:collect_os).and_return(:aix)
+        allow(@plugin).to receive(:shell_out).with("uname -L").and_return(mock_shell_out(0, "29 l273pp027", nil))
+        allow(@plugin).to receive(:shell_out).with("uname -W").and_return(mock_shell_out(0, "0", nil))
+        allow(@plugin).to receive(:shell_out).with("lswpar -L").and_return(mock_shell_out(0, @lswpar_L, nil))
+        @plugin.run
```

Comment:
`plugin` is available to use here and it has the same three `shell_out` stubs on it: https://github.com/chef/ohai/blob/if/fix-aix-attrs/spec/unit/plugins/aix/virtualization_spec.rb#L240-L248

You can replace `@plugin` with `plugin` and remove 257-261.